package.json

```diff
@@ -7,9 +7,10 @@
     "lerna-changelog": "cli.js"
   },
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/__mocks__/GithubAPI.js,src/__mocks__/Changelog.js",
```

Comment:
I recommend using them within one project but I do not recommend publishing them to npm.